src/mlpack/tests/main_tests/adaboost_test.cpp

```diff
@@ -398,8 +435,9 @@ BOOST_AUTO_TEST_CASE(AdaBoostDiffTolTest)
 
   // Calculate accuracy.
   arma::Row<size_t> output;
+  arma::mat probabilities;
   CLI::GetParam<AdaBoostModel*>("output_model")->Classify(testData,
-       output);
+       output, probabilities);
```

Comment:
No need to use the overload with `probabilities` here since the code is not checking anything about `probabilities` later. :+1: